src/go/privileged-service/pkg/port/server.go

```diff
@@ -17,83 +17,93 @@ limitations under the License.
 package port
 
 import (
-	"errors"
+	"encoding/json"
 	"fmt"
-	"io"
 	"net"
 
+	"github.com/Microsoft/go-winio"
+	"github.com/pkg/errors"
```

Comment:
Why are we switching to `pkg/errors`? I thought the stdlib one is used these days.
`errors.Wrap(err, "foo")` can be replaced with `fmt.Errorf("foo: %w", err)`.